test/libsolidity/util/SoltestTypes.h

```diff
@@ -193,6 +197,9 @@ struct FunctionCallExpectations
 	/// A Comment that can be attached to the expectations,
 	/// that is retained and can be displayed.
 	std::string comment;
+	/// An expectation can also be defined by a builtin function.
+	std::shared_ptr<FunctionCall> builtin;
```

Comment:
Can you give an example for this?

Reply:
This looks very complicated. Why is the expectation of a builtin not the same as a regular contract funciton call?